tests/testhelper.py

```diff
@@ -255,11 +256,11 @@ def wait_for_ajax(self):
             pass
 
     @staticmethod
-    def selenium_wrapper(func, *args):
+    def selenium_wrapper(func):
```

Comment:
yeah, that wrapper prob has to be redone where we past things independently and then re-assemble to do it in the wrapper since you cant pass the function with params like that. or pull the retry logic out of a wrapper and do it around each call but then thats a lot of bloat

Reply:
ok i think i might have a solution, just need to re-test everything

Reply:
Was just about to write you that the fix is easy, just use `*args` and pass the arguments. I was just about to apply it, but will wait for you.

Reply:
was a bit more than that, forced pushed update so you can see changes. i did drop one wrapper for `self.driver.switch_to.alert.dismiss()` that you had as none of the others ones had it and it was causing a hiccup due to the exception not passing if triggered for it..